src/commands/hld/reconcile.ts

```diff
@@ -70,8 +72,16 @@ export const execAndLog = async (commandToRun: string): Promise<ExecResult> => {
   const pipeOutputToCurrentShell = true;
   const result = await exec(commandToRun, pipeOutputToCurrentShell);
   if (result.error) {
-    logger.error(`an error occurred executing command: \`${commandToRun}\``);
-    throw result.error;
+    const errorInfo = buildError(
+      errorStatusCode.CMD_EXE_ERR,
+      {
+        errorKey: "hld-reconcile-err-cmd-failed",
+        values: [commandToRun],
+      },
+      result.error
+    );
+    logError(errorInfo);
```

Comment:
i think we do not need to logError because this error will appear at the end as a exception chain